test/ejabberd_tests/tests/distributed_helper.erl

```diff
@@ -43,42 +42,58 @@ wait_until_started(Cmd, Retries) ->
             ok;
         _ ->
             timer:sleep(1000),
-            wait_until_started(Cmd, Retries-1)
+            wait_until_started(Cmd, Retries - 1)
     end.
 
 wait_until_stopped(_, 0) ->
     erlang:error({timeout, stopping_node});
 wait_until_stopped(Cmd, Retries) ->
     case os:cmd(Cmd) of
-        "pong" ++ _->
+        "pong" ++ _ ->
             timer:sleep(1000),
-            wait_until_stopped(Cmd, Retries-1);
+            wait_until_stopped(Cmd, Retries - 1);
         _ ->
             ok
     end.
 
 verify_result(Op) ->
-    Node1 = ct:get_config(ejabberd_node),
-    Node2 = ct:get_config(ejabberd2_node),
+    Node1 = mim(),
+    Node2 = mim2(),
     DbNodes1 = rpc(Node1, mnesia, system_info, [running_db_nodes]),
     DbNodes2 = rpc(Node2, mnesia, system_info, [running_db_nodes]),
     Pairs = [{Node1, DbNodes2, should_belong(Op)},
-             {Node2, DbNodes1, should_belong(Op)},
-             {Node1, DbNodes1, true},
-             {Node2, DbNodes2, true}],
+        {Node2, DbNodes1, should_belong(Op)},
+        {Node1, DbNodes1, true},
+        {Node2, DbNodes2, true}],
     [?assertEqual(ShouldBelong, lists:member(Element, List))
-     || {Element, List, ShouldBelong} <- Pairs].
+        || {Element, List, ShouldBelong} <- Pairs].
 
-should_belong(add)    -> true;
+should_belong(add) -> true;
 should_belong(remove) -> false.
 
 cluster_op_timeout() ->
     %% This timeout is deliberately a long one.
-    timer:seconds(15).
+    timer:seconds(30).
 
 rpc(Node, M, F, A) ->
     rpc(Node, M, F, A, timer:seconds(5)).
 
 rpc(Node, M, F, A, TimeOut) ->
     Cookie = ct:get_config(ejabberd_cookie),
     escalus_ct:rpc_call(Node, M, F, A, TimeOut, Cookie).
+
+get_or_fail(Key) ->
+    Val = ct:get_config(Key),
+    Val == undefined andalso error({undefined, Key}),
+    Val.
+
+
+%% MongooseIM node names
+mim() ->
```

Comment:
I'd move this function  (also mim2 and fed) to `ejabberd_node_utils`. IMO they better suite there. Ideally would be to rename the `ejabberd_node_utils` to  `mongoose_node_utils`, or just `node_utils`. But this may introduce too many changes for this PR so maybe it's a subject of a separate PR.